.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: python
```

Comment:
Let's remove Travis. Let's have a sync to fall back to codeclimate

Reply:
The blocker I was facing is because I failed to use circleci to send the test data to Codeclimate(to setup the test coverage badge) or coveralls. That's why I decided to use Travis CI to help with test coverage. Maybe we discuss this further during the standups